pkg/promotion/promotion.go

```diff
@@ -24,6 +25,32 @@ func PromotesImagesInto(configSpec *cioperatorapi.ReleaseBuildConfiguration, pro
 	return !isDisabled(configSpec) && promotionNamespace == extractPromotionNamespace(configSpec)
 }
 
+// AllPromotionImageStreamTags returns a set of all ImageStreamTags this config promotes to.
+func AllPromotionImageStreamTags(configSpec *cioperatorapi.ReleaseBuildConfiguration) sets.String {
+	result := sets.String{}
+
+	if isDisabled(configSpec) {
+		return result
+	}
+
+	namespace := extractPromotionNamespace(configSpec)
+	name := extractPromotionName(configSpec)
+
+	if namespace == "" || name == "" {
```

Comment:
It is good to have this check here with its unittest cases included, but I don't think that we will ever hit this. There is already a validation in the ci-operator config and I am pretty sure that a configuration that misses those fields in the promotion will never exist in the release repo because the presubmits won't pass (finger crossed, maybe its a good idea to double check this.)

Reply:
Can you approve the PR if its fine?